web/themes/custom/novel/novel.theme

```diff
@@ -326,3 +326,29 @@ function novel_preprocess_views_view_unformatted(&$variables): void {
     }
   }
 }
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function novel_preprocess_input__submit(array &$variables): void {
+  $original_classes = $variables['attributes']['class'];
+
+  if (in_array('webform-button--submit', $original_classes)) {
+    $variables['attributes']['class'] += array_merge($original_classes, [
```

Comment:
The `+=` is not needed since `array_merge` is used.
So I think it should be:
```php
    $variables['attributes']['class'] = array_merge($original_classes, [
```